docs/reference/configuration.md

```diff
@@ -204,6 +204,9 @@ This reference uses `.` to denote the nesting of values.
 * `plugins.default_path` `(string : ~/.lakefs/plugins)` - Absolute path to the root of lakeFS's plugins location.
 * `plugins.properties.<plugin name>.path` `(string : )` - Absolute path to the location of `<plugin name>`'s binary location.
 * `plugins.properties.<plugin name>.version` `(uint : )` - Version of the `<plugin name>` plugin. The version must be > 0.
+* `installation.user_name` `(string : )` - When specified, an initial admin user will be created when the server is first run. Works only when `database.type` is set to local. Requires `setup.access_key_id` and `setup.secret_access_key`. 
```

Comment:
@nopcoder, noticed this change in the release of `v0.105.0` and I suspect the documentation of this one is incorrect?

Documentation now says: "Requires `setup.access_key_id` and `setup.secret_access_key`."

But, should this not instead say: "Requires `installation.access_key_id` and `installation.secret_access_key`."?

Reply:
Right - Thank you! I will push a fix by next week (PTO)